bin/node-template/shell.nix

```diff
@@ -0,0 +1,35 @@
+let
```

Comment:
We don't need `node-template`

Reply:
[Removed in v0.9.16](https://github.com/Cerebellum-Network/pos-network-node/pull/165/commits/574a020bcef14992638e944489806ebc59b1cb83)